props/jetpack_Oli_buttons.h

```diff
@@ -0,0 +1,931 @@
+        return true;
+      // Jetpack mishaps:
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_HELD_MEDIUM, MODE_OFF):
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_HELD_MEDIUM, MODE_ON):
+        JetpackMishaps();
+        return true;
+      // Missile Launch:
+      case EVENTID(BUTTON_POWER, EVENT_SECOND_SAVED_CLICK_SHORT, MODE_OFF):
+      case EVENTID(BUTTON_POWER, EVENT_SECOND_SAVED_CLICK_SHORT, MODE_ON):
+        missilelaunch_ = true;
+        return true;
+
+      // Abort missile sequence, maybe? ******** only for (2 & 3 buttons while off): ********
+      case EVENTID(BUTTON_AUX2, EVENT_FIRST_CLICK_LONG, MODE_OFF):
+      case EVENTID(BUTTON_AUX, EVENT_FIRST_CLICK_LONG, MODE_OFF):
+        missilelaunch_ = false;
+        state_machine_.reset_state_machine();
+        PVLOG_NORMAL << "Missile Sequence Aborted.\n";
+        SaberBase::DoEffect(EFFECT_DISARM,0);
+        return true;
+
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_CLICK_LONG, MODE_OFF):
+      // Next Preset (1 button):
+#if NUM_BUTTONS == 1
+        if (!mode_volume_) next_preset();
+        return true;
+#else
+        // Start or Stop Track (2 and 3 buttons):
+#ifdef JETPACK_TRACK_PLAYER
+        if (!istrack_playing_) StartTrackPlayer();
+        else StopTrackPlayer();
+#else
+        StartOrStopTrack();
+#endif
+        return true;
+#endif // NUM_BUTTONS == 1
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_CLICK_LONG, MODE_ON):
+      // Volume Down (1 button):
+        if (mode_volume_) VolumeDown();
+        return true;
+
+      // Next Preset and Volume Down (2 and 3 buttons):
+      case EVENTID(BUTTON_AUX, EVENT_FIRST_CLICK_SHORT, MODE_OFF):
+        if (!mode_volume_) next_preset();
+        else VolumeDown();
+        return true;
+      case EVENTID(BUTTON_AUX, EVENT_FIRST_CLICK_SHORT, MODE_ON):
+        if (mode_volume_) VolumeDown();
+        return true;
+
+#if NUM_BUTTONS >= 3
+      // Previous Preset (3 buttons):
+      case EVENTID(BUTTON_AUX2, EVENT_FIRST_CLICK_SHORT, MODE_OFF):
+#else
+      // Previous Preset (1 and 2 buttons):
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_HELD_LONG, MODE_OFF):
+#endif // NUM_BUTTONS >= 3
+        if (!mode_volume_) previous_preset();
+        return true;
+
+      // Activate Muted (Turn's Saber ON Muted):
+      case EVENTID(BUTTON_POWER, EVENT_SECOND_HELD, MODE_OFF):
+        if (SetMute(true)) {
+          unmute_on_deactivation_ = true;
+          if (!hybrid_font.PlayPolyphonic(&SFX_mute)) { // was announcemode(&SFX_mute);
+            // Use beeper for fallback sounds
+            beeper.Beep(0.05, 2000);
+            beeper.Silence(0.05);
+            beeper.Beep(0.05, 2000);
+          }
+          FastOn();
+        }
+        return true;
+
+      // Color Change mode (1 button):
+#if NUM_BUTTONS == 1
+      case EVENTID(BUTTON_POWER, EVENT_THIRD_SAVED_CLICK_SHORT, MODE_ON):
+#ifndef DISABLE_COLOR_CHANGE
+        ToggleColorChangeMode();
+#endif
+        return true;

```

Comment:
You probably shouldn't return true here if DISABLE_COLOR_CHANGE is defined, since it doesn't actually do anything in that case.